python/ray/serve/tests/test_metrics.py

```diff
@@ -32,7 +33,7 @@
 TEST_METRICS_EXPORT_PORT = 9999
 
 
-@pytest.fixture
+@pytest_asyncio.fixture
```

Comment:
I think we usually use `pytest.mark.asyncio`

Reply:
Yea, I think `pytest.fixture` getting deprecated in some future version. Saw this warning message in one of the failed run https://buildkite.com/ray-project/premerge/builds/29448#0192100b-ce04-4360-bcb2-a33f09cc1442/6-8202. But I can revert if we want to follow up to clean all the usage of `pytest.fixture` when that comes